src/site/markdown/payload_compression.md

```diff
@@ -0,0 +1,40 @@
+# Payload compression
+
+RPM packages can be compressed using one of several methods. Most of these methods allow setting a custom
+compression level.
+
+## Configuration
+
+Payload compression is configured using the `<payloadCoding>` and `<payloadFlags>`elements:
+
+```xml
+    <!-- These are the default values, both of which are optional -->
+    <configuration>
+      <payloadCoding>zstd</payloadCoding>
+      <payloadFlags>19</payloadFlags>
+    </configuration>
+```
+
+### Payload coding
+
+Payload coding must be one of the following compressors:
+
+* none (uncompressed)
+* gzip
+* lzma
+* bzip2
+* zstd (default compressor, with a compression level of 19)
+* xz
+
+### Payload flags
+
+In the RPM format, the payload flags are a string of arguments that are passed to the command-line compressor. Since
+RPM Builder uses Java and not the command line, the flags currently supported consist of a single integer value. This
+value is used as the compression level if the compressor supports it.
+
+The compressor `none` does not support any flags.
+
+The compressors `gzip`, `lzma`, `bzip2`,and `xz` support a range of compression levels between 0 (no compression) and 9.
+Additionally, `gzip` supports the value -1 for the default compression level.
+
+The compressor `zstd` supports a range of compression levels between 0 (default compression level) and 22.
```

Comment:
Doesn't there need to be an addition to `src/site/site.xml` to add this to the menu?